sass/directives/_visual-guides.scss

```diff
@@ -0,0 +1,118 @@
+/* A simple, center dividing column guide / line. See /recruiting-solutions/small-business-subscription-plans-and-pricing
+ * The guide can extend out of its relatively positioned parent container on the bottom and top
+ * This guide does not include steps. It is only a center line.
+ * You must have a parent container (relatively positioned) for this to work properly.
+ */
+@mixin visual-guide-divider($extend-top: 0px, $extend-bottom: 0px) {
+
+  @if $extend-top != 0px {
+    bottom: -($extend-bottom);
+    top: -($extend-top);
+  } @else {
+    bottom: $extend-bottom;
+    top: $extend-top;
+  }
+
+  left: 0;
+  margin: auto;
+  position: absolute;
+  right: 0;
+  text-align: center;
+  width: 50px;
+
+  &:before, &:after {
+    content: '';
+    background-color: white;
+    border: $base-border;
+    border-radius: 7px; //TODO: New border radius variable?
+    display: block;
+    height: 14px;
```

Comment:
height and width could be a variable so it's easier to change them at the same time, or you could just move width up, if they're next to each other i think that's easy enough